pkg/plugins/policies/meshratelimit/api/v1alpha1/meshratelimit.go

```diff
@@ -40,7 +40,7 @@ type From struct {
 type To struct {
 	// TargetRef is a reference to the resource that represents a group of
 	// clients.
-	TargetRef common_api.TargetRef `json:"targetRef"`
+	TargetRef common_api.TargetRef `json:"targetRef,omitempty"`
```

Comment:
no need for `omitempty` here, you probably missed this one when reverted the optional change